rules/falco_rules.yaml

```diff
@@ -1396,15 +1397,17 @@
 
 - rule: Read sensitive file trusted after startup
   desc: >
-    an attempt to read any sensitive file (e.g. files containing user/password/authentication
+    An attempt to read any sensitive file (e.g. files containing user/password/authentication
     information) by a trusted program after startup. Trusted programs might read these files
-    at startup to load initial state, but not afterwards.
+    at startup to load initial state, but not afterwards. Can be customized as needed.
   condition: sensitive_files and open_read and server_procs and not proc_is_new and proc.name!="sshd" and not user_known_read_sensitive_files_activities
   output: >
-    Sensitive file opened for reading by trusted program after startup (user=%user.name user_loginuid=%user.loginuid
-    command=%proc.cmdline pid=%proc.pid parent=%proc.pname file=%fd.name parent=%proc.pname gparent=%proc.aname[2] container_id=%container.id image=%container.image.repository)
+    Sensitive file opened for reading by trusted program after startup (user=%user.name uid=%user.uid user_loginuid=%user.loginuid 
+    process=%proc.name proc_exepath=%proc.exepath command=%proc.cmdline pid=%proc.pid parent=%proc.pname 
+    file=%fd.name gparent=%proc.aname[2] ggparent=%proc.aname[3] gggparent=%proc.aname[4] terminal=%proc.tty 
+    container_id=%container.id image=%container.image.repository namespace=%k8s.ns.name pod_name=%k8s.pod.name)
```

Comment:
This one did sound a little bit noisy but I see that the list of programs that are involved is limited (server, db, ...). Makes sense to me.